File: CCC Python/2018/J1.py
digitOne = int(input())
digitTwo = int(input())
digitThree = int(input())
digitFour = int(input())

if digitOne == 8 or digitOne == 9:
    if digitFour == 8 or digitFour == 9:
        if digitTwo == digitThree:
            print("ignore")
        else:
            print("answer")
    else:
        print("answer")
else:
    print("answer")

File: CCC C++/2018/J1.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main(){
    int digitOne;
    int digitTwo;
    int digitThree;
    int digitFour;

    scanf("%d", &digitOne);
    scanf("%d", &digitTwo);
    scanf("%d", &digitThree);
    scanf("%d", &digitFour);

    if (digitOne == 8 || digitOne == 9){
        if (digitFour == 8 || digitFour == 9){
            if (digitTwo == digitThree){
                printf("ignore");
            }
            else{
                printf("answer");
            }
        }
        else{
            printf("answer");
        }
    }
    else{
        printf("answer");
    }

}
